plugins/com.xored.q7.quality.mockups.swt/src/org/eclipse/swt/button/Button_Radio.java

```diff
@@ -37,7 +43,24 @@ public Control construct(Composite parent) {
 		
 		radioButtons[4] = new Button(composite, SWT.RADIO);
 		radioButtons[4].setToolTipText("Choice 4");
+
+		radioButtons[5] = new Button(composite, SWT.RADIO);
+		radioButtons[5].setText("Choice 5");
+
+		radioButtons[5].addSelectionListener(new SelectionAdapter() {
+            @Override
+            public void widgetSelected(SelectionEvent e) {
+                selectionChange();
+            }
+        });
 		
 		return null;
 	}
+
+	private void selectionChange() {
+        boolean buttonSelected = radioButtons[5].getSelection();
+        radioButtons[1].setEnabled(!buttonSelected);

```

Comment:
Unintended side-effect. We are testing, that a button received deselection event, if listener itself produces more events, this may lead to false negatives.

Reply:
fixed